packages/ocaml-protoc/ocaml-protoc.2.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Protobuf compiler for OCaml"
+description: "Protobuf compiler for OCaml"
+maintainer: "Maxime Ransan <maxime.ransan@gmail.com>"
+authors:[
+  "Maxime Ransan <maxime.ransan@gmail.com>"
+]
+homepage: "https://github.com/mransan/ocaml-protoc"
+bug-reports:"https://github.com/mransan/ocaml-protoc/issues"
+dev-repo:"git+https://github.com/mransan/ocaml-protoc.git"
+license: "MIT"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>="4.02.1"}
+  "dune"  {build}
+  "stdlib-shims"
+  "ocamlbuild" {build}
```

Comment:
```suggestion
```
This packages doesn't seem to depend on ocamlbuild anymore

Reply:
unfortunately, the following [package](https://opam.ocaml.org/packages/ocamlbuild-protoc/) has a transitive dependency on ocamlbuild through ocaml-protoc. Here is the  [issue](https://github.com/Khady/ocamlbuild-protoc/issues/1). The short term fix is to leave the ocamlbuild dependency in this package pending the resolution. 

(see my previous publishing attempt [here](https://github.com/ocaml/opam-repository/pull/15204)